docs/DeveloperGuide.adoc

```diff
@@ -493,340 +503,226 @@ Here are the steps to create a new release.
+for reference
 
-**Main:**
+=== Foo Yong Jie
+Major enhancement: Improving the UI / UX of the application, including implementing separate panels for tags, cards,
+and the display of cards.
+This allows users to use the application efficiently.
 
-. Add a `RemarkCommand` that extends link:{repoURL}/src/main/java/seedu/address/logic/commands/UndoableCommand.java[`UndoableCommand`]. Upon execution, it should just throw an `Exception`.
-. Modify link:{repoURL}/src/main/java/seedu/address/logic/parser/AddressBookParser.java[`AddressBookParser`] to accept a `RemarkCommand`.
+Minor enhancement: Implementing the feature to switch the theme of the application between light and dark so users
+can change it to suit their preference.
 
-**Tests:**
+=== Wong Ding Feng
+Major enhancement: Implementing a scheduler to prompt users which card they should review daily so that they are able to prioritise what to
+go through first.
 
-. Add `RemarkCommandTest` that tests that `executeUndoableCommand()` throws an Exception.
-. Add new test method to link:{repoURL}/src/test/java/seedu/address/logic/parser/AddressBookParserTest.java[`AddressBookParserTest`], which tests that typing "remark" returns an instance of `RemarkCommand`.
+Minor enhancement: Sorting of cards based on tags.
 
-===== [Step 2] Logic: Teach the app to accept 'remark' arguments
-Let's teach the application to parse arguments that our `remark` command will accept. E.g. `1 r/Likes to drink coffee.`
+=== Shawn Chua
+Major enhancement: Implementing polymorphic cards to include MCQ-type and fill-in-the-blanks-type cards so that users
+are able to memorise information in different ways.
 
-**Main:**
+Minor enhancement: Implementing editing of existing cards so that users are able to make changes should they
+make a mistake.
 
-. Modify `RemarkCommand` to take in an `Index` and `String` and print those two parameters as the error message.
-. Add `RemarkCommandParser` that knows how to parse two arguments, one index and one with prefix 'r/'.
-. Modify link:{repoURL}/src/main/java/seedu/address/logic/parser/AddressBookParser.java[`AddressBookParser`] to use the newly implemented `RemarkCommandParser`.
+[appendix]
+== User Stories
 
-**Tests:**
+Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-. Modify `RemarkCommandTest` to test the `RemarkCommand#equals()` method.
-. Add `RemarkCommandParserTest` that tests different boundary values
-for `RemarkCommandParser`.
-. Modify link:{repoURL}/src/test/java/seedu/address/logic/parser/AddressBookParserTest.java[`AddressBookParserTest`] to test that the correct command is generated according to the user input.
+[width="100%",cols="22%,<23%,<25%,<30%",options="header",]
+|=======================================================================
+|Priority |As a ... |I want to ... |So that I can...
+| `* *` | University Student | Create a new card                                        | I can review new material
```

Comment:
No "must have" user stories? I feel like adding and reviewing cards at least are quite important in a flash card application...